backend/backend.go

```diff
@@ -555,6 +557,38 @@ func (backend *Backend) Coin(code coinpkg.Code) (coinpkg.Coin, error) {
 	return coin, nil
 }
 
+func (backend *Backend) pollETHBalances() {
```

Comment:
This loop runs forever - should exit it upon backend.Close(). You would probably need to introduce quit channel, similar to https://github.com/BitBoxSwiss/block-client-go/pull/8.